pkg/controller/scyllacluster/sync_statefulsets.go

```diff
@@ -311,7 +310,12 @@ func (scc *Controller) beforeNodeUpgrade(ctx context.Context, sc *scyllav1.Scyll
 }
 
 func (scc *Controller) afterNodeUpgrade(ctx context.Context, sc *scyllav1.ScyllaCluster, sts *appsv1.StatefulSet, ordinal int32, services map[string]*corev1.Service) error {
-	host, err := controllerhelpers.GetScyllaHost(sts.Name, ordinal, services)
+	svcName := fmt.Sprintf("%s-%d", sts.Name, ordinal)
```

Comment:
naming.MemberServiceName?

Reply:
`naming.MemberServiceName` takes a rack spec, so I'd have to get the rack name from the statefulset, get the rack spec from SC to then get the statefulset name from rack inside the func, it seems completely redundant to me.

Reply:
no worries, seems like some of our helpers take bigger scope then they need